requirements-test.txt

```diff
@@ -0,0 +1,4 @@
+-r requirements.txt
+autopep8
+flake8
+wheel  # needed by autopep8
```

Comment:
I thought the point of requirements was to pull in the recursive dependencies. But if we need it, I suppose it is good to be explicit.
I'm a bit surprised that autopep8 depends on wheel.

Reply:
I'm also surprised about that building part...

Reply:
I suspect the problem is because those projects don't declare these dependencies explicitly